docs/src/content/docs/zh/showcase.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Starlight Showcase
```

Comment:
Noticing that this isn’t translated here or in the sidebar config:
<img width="299" alt="image" src="https://github.com/withastro/starlight/assets/357379/2f78ba1f-dd23-472e-9746-f09df6085656">
Is that intentional?

Reply:
Yes, I don't think `showcase` needs to be translated. In China, we also use the word `showcase` in our work.